### DISCUSSION TITLE
clock not syncronized or other error

### DISCUSSION THREAD

**Discussion by soulantea** [NightscoutFoundation/xDrip#2036] [Q&A]
Hi, have had an issue for last 5-6 hours, 

everything wroked fine for last 2 days, 

was in another room for a couple hours and somehow my sensor was stopped, im assume my kid accidently done it, 

any ways have tried to restart, disconnecting for 20 mins and re insterting it, but since it was my first sensor i threw away the code, couldnt find any info on how to reconnect it, 

so far 

unstalled re installed xdrip, inserted new sensor it let me do a warm up then hanged nothing not even stoped, 

gone throught settings found on 

https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html

and done 

https://navid200.github.io/xDrip/docs/Restart-G6-sensor.html

still nothing.

so still unsure what to do.

![Screenshot_2022-03-06-01-42-10-95_2dd2c1bcb8f659cd696bcba2072f751b](https://user-images.githubusercontent.com/70943212/156888148-74234b17-7072-4f1b-af50-433c5aa754c9.jpg)
![Screenshot_2022-03-06-01-35-51-55_2dd2c1bcb8f659cd696bcba2072f751b](https://user-images.githubusercontent.com/70943212/156888151-982fa50b-e508-4acc-a7c1-0a9e17a9e7d9.jpg)
![Screenshot_2022-03-06-01-35-44-40_2dd2c1bcb8f659cd696bcba2072f751b](https://user-images.githubusercontent.com/70943212/156888147-71b6ca41-25ed-4b28-be53-0892d6f9291a.jpg
![Screenshot_2022-03-06-01-45-19-55_2dd2c1bcb8f659cd696bcba2072f751b](https://user-images.githubusercontent.com/70943212/156888325-593a50aa-df1f-45d2-b02c-f0d52a83f0f6.jpg)
![Screenshot_2022-03-06-09-19-28-54_2dd2c1bcb8f659cd696bcba2072f751b](https://user-images.githubusercontent.com/70943212/156901644-e75a3c97-5a69-4f7c-b6a2-a925e8e464f5.jpg)
![Screenshot_2022-03-06-09-42-05-79_2dd2c1bcb8f659cd696bcba2072f751b](https://user-images.githubusercontent.com/70943212/156902121-de8f0335-524d-43f9-a4dd-7423d0c02af5.jpg)
![Screenshot_2022-03-06-11-58-07-74_2dd2c1bcb8f659cd696bcba2072f751b](https://user-images.githubusercontent.com/70943212/156904752-9e2dec0d-4894-4c6a-979b-1c4fd513f0ef.jpg)
)

**Comment by Navid200** [comment]
Would you please post a screenshot of the system status page?  You have posted the classic status page.  Please also post the G5/G6 status page.
You can blackout the serial number if you like.  But, please nothing else on that page.

**Reply by soulantea** [reply]
have updated. took out the transmitter over night and same error when i setup again

**Reply by Navid200** [reply]
It's pointless to start sensor as long as there is no connectivity.  if there was proper connectivity, you would see the battery voltages on the system status page.
What's surprising is that it says last connect 1 minute ago.

Does last connected keep going up and roll back to 0 just before reaching 5 minutes?  Or, has it already gone over 5 minutes?

**Reply by soulantea** [reply]
Just checked it was last connected a minute ago i just used recommend setting after setting up. Will post a new pic

**Reply by Navid200** [reply]
Looks like voltages are shown now.
If "Last connected" is consistently less than 5 minutes, you can now perform a restart, following instructions, and it should restart properly.
Of course, please pay attention to the warnings as the readings will be in error after a restart and you will need to calibrate following instructions.
https://navid200.github.io/xDrip/docs/Restart-G6-sensor.html

**Reply by soulantea** [reply]
done this, and this is a brand new sensor cause my first one of 3 days stopped. ill have another go.

**Reply by Navid200** [reply]
It doesn't matter.  If you don't wait long enough for a new transmitter to establish proper connectivity, your sensor start will fail.  If you then restart when the voltages are not shown, meaning still have not established connectivity, it will still fail.

**Reply by soulantea** [reply]
i herd if you restart the transmitter it brakes it? im trying different option unstianlling re installing and going through again, i got it the pooint of seeing the voltage but the tranmitter status was unkowen so i did a complete reinstall, and still going around in circles.

**Reply by Navid200** [reply]
You are wasting your time if you believe that you can break the transmitter by restarting the sensor.
I have explained the risk of restarting the sensor clearly.  It's the inaccuracy caused by the interference with factory calibration.

**Reply by Navid200** [reply]
Not sure what you mean by restarting the "transmitter".

**Reply by soulantea** [reply]
i installed the build your own decom to test and said to replace both sensor and transmitter so i got a new one, working fine with dexcom, want it to work with dexcom but ill wait for another stabel build as i think i may of broke it.
